addons/point_of_sale/views/point_of_sale_dashboard.xml

```diff
@@ -105,6 +105,8 @@
                                 <div class="o_kanban_card_header_title mb16">
                                     <div class="o_primary o_kanban_card_header_title_name">
                                         <t t-esc="record.name.value"/>
```

Comment:
instead of having a space before `(Closed)` you can change this `t` tag into a `span` and add a `me-1` class to it.
Adding an actual space character works, but it's fragile